src/pages/bug-bounty/vulnerabilities-types/index.tsx

```diff
@@ -70,81 +70,93 @@ const StyledLi = styled.li`
     }
 `
 
-const VulnerabilitiesTypes = () => {
-    return (
-        <Layout type="security">
-            <SEO
-                title={localize('Types of vulnerabilities')}
-                description={localize('Types of vulnerabilities')}
-            />
-            <Hero />
-
-            <StyledContainer direction="column" align="start">
-                <Header type="heading-2" align="center">
-                    {localize('Types of vulnerabilities')}
-                </Header>
-                <div>
-                    {vulnerability_types_content.map((item, idx) => {
+const VulnerabilitiesTypes = () => (
+    <Layout type="security">
+        <SEO
+            title={localize('Types of vulnerabilities')}
+            description={localize('Types of vulnerabilities')}
```

Comment:
Shouldn't we refactor this as well by new approach?
<img width="1489" alt="Screenshot 2023-01-18 at 13 15 49" src="https://user-images.githubusercontent.com/103182473/213145503-8b92f99d-5fa4-41b9-9a1e-d4c494b6f22e.png">